custom/conf/app.example.ini

```diff
@@ -1143,6 +1143,83 @@ LEVEL = Info
+;;                       where {column} is a column title
+;;                       (default value is context=project)
+;;
+;; Commands can have arguments using the command:argument syntax.
+;; They are available both as a trigger and as an action.
+;; Available commands:
+;; - move:{column}   where {column} is a column title
+;; - status:{status} where {status} is one of: closed, reopened
+;; - assign_project  assigns the current project to the issue/pr
+;; - assign_reviewer adds the current user as a requested reviewer
+;; - assign          assigns the current user to the issue/pr
+;;                   when used as a trigger, it triggers when issue/pr is no longer unassigned
+;; - unassign        unassign the current user from the issue/pr
+;;                   when used as a trigger, it triggers when issue/pr becomes unassigned
+;; - approve         approves a pull request (default target=pr)
+;; - unapprove       unapproves a pull request (default target=pr)
+;; - xref:{action}   a cross-reference is made between issue and pull request
+;;                   where {action} is one of: none, closes, reopens, neutered
+;;                   When xref is used as a trigger, target=pr can target a
+;;                   pull request which will close an issue in the current project,
+;;                   even if that pull request is not currently assigned to the project.
+;;                   (default target=issue)
+;; - label:{label}   adds a label to the issue/pr, where {label} is a label accessible in the current project
+;; - unlabel:{label} remove label from the issue/pr
+;RULE1  = trigger=status:closed,    action=move:Done
+;RULE2  = trigger=status:reopened,  action=move:To Do
+;RULE3  = trigger=approve,          action=move:Done,      target=pr, context=column:In Progress
+;RULE4  = trigger=xref:closes,      action=assign_project, target=pr
+;RULE5  = trigger=assign_project,   action=move:To Do,     target=pr
+;RULE6  = trigger=move:To Do,       action=unassign
+;RULE7  = trigger=move:To Do,       action=status:reopened
+;RULE8  = trigger=move:To Do,       action=clear_reviewers
+;RULE9  = trigger=move:In Progress, action=assign
+;RULE10 = trigger=move:In Progress, action=status:reopened
+;RULE11 = trigger=move:In Progress, action=assign_reviewer
+;RULE12 = trigger=move:Done,        action=status:closed,          target=issue
+;RULE13 = trigger=move:Done,        action=approve,                target=pr
+;RULE14 = trigger=assign,           action=move:In Progress,       target=issue, context=column:To Do
+;RULE15 = trigger=assign_reviewer,  action=move:In Progress,       target=pr,    context=column:To Do
+;RULE16 = trigger=xref:closes,      action=label:Status/Blocked,   target=issue, context=column:In Progress
+;RULE17 = trigger=approve,          action=unlabel:Status/Blocked, target=issue, context=column:In Progress
+;RULE18 =
+;RULE19 =
+;RULE20 =
+;RULE21 =
+;RULE22 =
+;RULE23 =
+;RULE24 =
+;RULE25 =
```

Comment:
As I think I've already mentioned, do not create a static amount of rules.
Instead, iterate over all keys in the section.

Reply:
You might have mentioned it, albeit encoded into a single `?` character in a previous comment.  Thus it might have flown over my head a bit and thank you for taking the time to write it out in words here.

I guess we could do that, but could not see that we did a similar thing elsewhere (iterating over dynamic keys in the config). Thus I decided to kick it off with the "stupidly simple" approach first.